docs/routers.md

```diff
@@ -0,0 +1,50 @@
+## Routers
+Gateway offers two entry-level router options, providing users with a straightforward and easy-to-use starting point for their routing needs.
+
```

Comment:
How about adding a title about what info is available for users to use when making a customized module